test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerTest.java

```diff
@@ -103,6 +103,9 @@ public void testCode() throws Exception {
 
                 String output = jedisOps.get(sessionToken);
                 assertNull("Should no longer be session data", output);
+                } catch(Exception e) {
+                    e.printStackTrace();
+                }
```

Comment:
Rethrowing (without the catch, you don't get a stack trace in the error logs, this seems to be something about the Play test infrastructure being used here).